addons/account/report/account_invoice_report.py

```diff
@@ -100,7 +100,7 @@ def _select(self):
                 -line.balance * currency_table.rate                         AS price_subtotal,
                 -COALESCE(
                    -- Average line price
-                   (line.balance / NULLIF(line.quantity, 0.0))
+                   (line.balance / NULLIF(line.quantity, 0.0)) * (CASE WHEN move.move_type = 'out_refund' THEN -1 ELSE 1 END)
```

Comment:
Missing `in_invoice` and `in_receipt` then

Also, why is it not done for `price_subtotal`?

Reply:
Hello, i tried to add this in the price_subtotal but i get weird output from the test https://github.com/odoo/odoo/blob/c99e305849f84810f5f95f2558028dcf04535825/addons/account/tests/test_account_invoice_report.py#L109
`-price_average:6.0
-price_subtotal:6.0
-quantity:1.0
+price_average:600
+price_subtotal:-600
+quantity:-1`
I don't get how adding a * -1 can turn 600 into 6

Reply:
@william-andre

Reply:
The full traceback might be worth checking.
Look at the `order` param in https://github.com/odoo/odoo/blob/c99e305849f84810f5f95f2558028dcf04535825/addons/account/tests/test_account_invoice_report.py#L100

It might be interresting